src/components/Button/Button.scss

```diff
@@ -266,49 +255,27 @@ $block: '.#{variables.$ns}button';
     &__text {
         display: inline-block;
         white-space: nowrap;
-        padding: 0 var(--g-button-padding, var(--_--padding));
     }
 
     &__icon {
-        display: inline-block;
-        position: relative;
-        width: var(--g-button-height, var(--_--height));
+        display: inline-flex;
+        justify-content: center;
+        width: var(--g-button-height, var(--_--icon-size));
         height: var(--g-button-height, var(--_--height));
         pointer-events: none;
 
-        &::after {
-            content: '\00a0';
```

Comment:
Seems like it didn't have any effect before

Reply:
Please rebase your branch, I added test with text interation.

Reply:
returned and fixed 85ec4446b414efc7d2038ff8efb5307cb5914587